docs/developers-docs/development-environment/backend_environment.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Backend environment
+summary: A guide of how to run a development environment for the backend
+layout: default
+status: Draft
+parent:  Development Environment
+grand_parent: Developers Guide
+nav_order: 2
+---
+
+# How to run a development environment for the backend
+{: .no_toc }
+
+This page describes a list of steps to follow to run a development environment for the backend locally
+{: .fs-6 .fw-300 }
+
+We assume in this guide:
+* That **you have access to a shell Terminal**, either because you are running on a Linux/Mac operating system or Windows wth [WSL 2](https://learn.microsoft.com/en-us/windows/wsl/) installed
+
+{: .important }
+You will use this same backend  environment for **ontologies_api**, **ontologies_linked_data**, and **goo** projects
+
+
+## Steps to follow
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+
+
+
+## Run the backend infrastructure using VirtualBox
+
+
+We will use a virtual machine with a **nat port forwarding** of all ports needed to run the `ontologies_api` project locally.
+Port forwarding will make them all accessible from the `localhost` 
+
+
+### Install the appliance on VirtualBox
+First get [the appliance ](/administration/steps/initial_installation)  and run it on VirtualBox
+
+### Configure port forwarding
+Then on VirtualBox go to Settings > Network, and change the adapter to `NAT` and click on `Advanced` then `Port Forwarding`. 
+Where you add the following ports to forward:
+
+![Backend infrastructure port forwarding]({{site.baseimgs}}/developers/virtual-box-nat-configuration.png)
```

Comment:
Mapping NAT ssh 22 to 22 don't worked for me. I used a different port for the host (2222).

Reply:
I think it didn't work for you because had already an ssh server running on your machine. 
I will add a note about this. To tell to use port 2222 if the 22 didn't work.